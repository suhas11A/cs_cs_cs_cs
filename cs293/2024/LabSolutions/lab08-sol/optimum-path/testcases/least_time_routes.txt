town_hall main_circle hospital school park offices mall
3.0 10.0 5.0 3.0 1.0 6.0 4.0
0 1 2.0
0 5 105.5 
1 3 7.2
2 6 5.0
3 4 1.2
3 2 9.5
5 4 8.6
3 6 33.7
6 5 11.1