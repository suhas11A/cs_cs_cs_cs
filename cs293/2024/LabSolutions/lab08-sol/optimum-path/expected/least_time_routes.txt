0
0 1
0 1 3 2
0 1 3
0 1 3 4
0 1 3 2 6 5
0 1 3 2 6